src/pool/rpc.h

```diff
@@ -323,6 +327,16 @@ CRT_RPC_DECLARE(pool_tgt_disconnect, DAOS_ISEQ_POOL_TGT_DISCONNECT,
 CRT_RPC_DECLARE(pool_tgt_query, DAOS_ISEQ_POOL_TGT_QUERY,
 		DAOS_OSEQ_POOL_TGT_QUERY)
 
+#define DAOS_ISEQ_POOL_TGT_FETCH_HDLS	/* input fields */	 \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
This is identical to other declarations in this file, no plans to change.